secrets/store.go

```diff
@@ -80,8 +80,10 @@ func (s *Store) getSecrets() *Secrets {
 // but can still access the secrets as they were before Close is called.
 //
 // It's OK to call Close multiple times. Calls after the first one are no-ops.
-func (s *Store) Close() {
+// Close doesn't return non-nil errors, but implements io.Closer
```

Comment:
Add a blank line before this line (so it's a new paragraph), and end it with period (`.`).